.python-version

```diff
@@ -0,0 +1,8 @@
+3.9.2
+3.8.8
+3.7.10
+3.6.13
+3.5.10
+2.7.18
+pypy-5.7.1
+pypy3.7-7.3.3
```

Comment:
This is a https://github.com/pyenv/pyenv config file which helps running isolated Python installs locally